Form/Type/SeoMetadataType.php

```diff
@@ -69,8 +76,15 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
      */
     public function configureOptions(OptionsResolver $resolver)
     {
+        $class = null;
+        if ($this->container->has('cmf_seo.backend_type_phpcr')) {
```

Comment:
hm. why not have a configuration option for the seo metadata class that defaults to the phpcr document if phpcr is active and to the model otherwise? and have the DI set that parameter on the container, and use constructor injection?